playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -234,20 +234,35 @@ export default class ParticipantListPage extends DsmPageBase {
     await waitForNoSpinner(this.page);
   }
 
+  /**
+   * Find a participant with the requested tab
+   * @param opts findPediatricParticipant - determines if an adult or child participant should be returned
+   * @param opts tab - the tab to be searched for e.g. onc history tab
+   * @param opts rgpProbandTab - determines if a rgp proband tab is being searched for
+   * @param opts uriString - the uri string to use when initially filtering for participants. Defaults to '/ui/applyFilter'
+   * @param opts prefix - the prefix or name of the test users to be used in the search e.g. E2E or KidFirst for playwright created participants. Defaults to 'E2E'
+   * @returns A participant with the requested tab
+   */
   async findParticipantWithTab(
```

Comment:
amount of parameters / optional parameters was getting large so added a comment for explanation